pkg/api/service/annotations.go

```diff
@@ -25,4 +25,9 @@ const (
 	//
 	// Not all cloud providers support this annotation, though AWS & GCE do.
 	AnnotationLoadBalancerSourceRangesKey = "service.beta.kubernetes.io/load-balancer-source-ranges"
+
+	// An annotation that denotes if this Service desires to only route to local
+	// endpoints. Typically used to preserve source IP in loadbalanced Services.
+	AnnotationOnlyNodeLocalEndpoints = "service.alpha.kubernetes.io/cloud-lb-only-node-local-endpoints"
```

Comment:
why include "cloud-lb" in the name of this annotation? one can use it for type=NodePort as well right?